charts/openshift-etcd-backup/templates/cronjob.yaml

```
@@ -11,6 +11,8 @@ spec:
       backoffLimit: 0
       template:
         spec:
+          securityContext:
+             {{- toYaml .Values.podSecurityContext | nindent 12 }}
```

One space too much here:
```suggestion
            {{- toYaml .Values.podSecurityContext | nindent 12 }}
```

---

wow, nice catch, thanks!